doc/oper.xml

```diff
@@ -1572,6 +1572,92 @@ gap> DigraphShortestDistance(D, [1, 3], [3, 5]);
 </ManSection>
 <#/GAPDoc>
 
+<#GAPDoc Label="Dominators">
+<ManSection>
+  <Oper Name="Dominators" Arg="digraph, root"/>
+  <Returns>A list of lists </Returns>
+    <Description>
+    <C>Dominators</C> is a function that takes a <A>digraph</A> and a
+    specific root <A>root</A> and returns the dominators of each vertex with
+    respect to the root. The root must be a verex of the digraph. The algorithm
+    is an implementation of the almost linear time algorithm by
+    Thomas Lengauer and Robert Endre Tarjan <Cite Key="LT79"/>. If there is
+    no path from the root to a specific vertex, the output will contain a
+    hole in the corresponding position. The complexity of this algorithm is 
+    <M>O(mlog n)</M> where <M>m</M> is the number of edges and <M>n</M> is the
```

Comment:
Lacking knowledge in complexity, but I was thinking that `DominatorTree` performs a dfs so it will have worst case complexity O(m + n), but `Dominators` only runs through the vertices that are reachable from the root vertex and so I cannot find a case, in which we would visit any of the edges not belonging in the subdigraph induced by the reachable nodes. Definitely not sure about this though!